modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -402,7 +402,7 @@ module FindPoaByParticipantId
       module VnpProcFormWebServiceBean
         DEFINITION =
           Bean.new(
-            path: 'VnpProcFormWebServiceBean',
+            path: 'VnpProcFormWebServiceBean/VnpProcFormService',
```

Comment:
Same [question](https://github.com/department-of-veterans-affairs/vets-api/pull/19893/files#r1892753872) as on #19893 - are we only adding the service to the path on Vnp beans?

Reply:
Resolved [here](https://github.com/department-of-veterans-affairs/vets-api/pull/19893#discussion_r1892906543).